packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initNotificationsMixin.ts

```diff
@@ -38,7 +38,7 @@ export const initNotificationsMixin = createSingletonMixin(
       #createNotification(type: Notification['type']) {
         return this.createNotification({
           mode: type,
-          duration: 3000,
+          duration: type === 'error' ? 0 : 3000,
```

Comment:
Move to duration to const, and the condition to an expression like `isStaticNotification`